recipes/weasyprint/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "WeasyPrint" %}
+{% set version = "v47" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

Comment:
version should probably not have the `v` prefix here.  I would recomend removing it above and adding the v to the url below

Reply:
You're right, I've changed this.